go.mod

```diff
@@ -6,7 +6,7 @@
 
 module k8s.io/kubernetes
 
-go 1.16
+go 1.17
```

Comment:
keep this at `go 1.16` for now, we can spawn a follow up issue to decide how to deal with the `tidy` changes in 1.17